src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    public String toString() {
+        return "Figure: rectangle, area: " + calculateArea() + " sq. units, "
+                + "length: " + length + " units, "
+                + "width: " + width + " units, "
+                + "color: " + getColor();

```

Comment:
Make sure to use English in your messages. The `toString()` method should return a message that is universal and consistent. If `Figure: rectangle` is not a placeholder and the message is actually hardcoded, consider using a constant or a way to retrieve this information programmatically to make it adaptable for other figures as well.